odk/extension/src/main/java/io/samagra/odk/collect/extension/handlers/ODKFormsHandler.kt

```diff
@@ -72,6 +72,35 @@ class ODKFormsHandler @Inject constructor(
         openForm(form, context)
     }
 
+
+    private fun checkForm(formId: String, tagValueMap: HashMap<String, String>){
```

Comment:
Hey @prabs3257 I see there is some code repetition in this function [here](https://github.com/Samagra-Development/odk-collect-extension/blob/3a0017b9ecfd3d8beafc2812d2f7f7eb1e24a52d/odk/extension/src/main/java/io/samagra/odk/collect/extension/handlers/ODKFormsHandler.kt#L201). Can you extract out the form files checking code into a single function and call that function both from prefillForm() and openForm().